### ISSUE TITLE
Web

### ISSUE THREAD

**Issue by winwisely99** [winwisely99/main#42]
We need to run as web because its important in case we loose app store access.

**Comment by winwisely99** [comment]
https://github.com/blankapp/flutter-widget-livebook
- interesting

many:
https://github.com/topics/flutter-web


Also there are routers and unlinsk that DO work with flutter web and mobile.
I will did them out.

**Comment by thevzurd** [comment]
Added web component as well. Need to customize the screens for web. These changes in UI will be similar to desktop

**Comment by thevzurd** [comment]
@winwisely99 reg : "FIrst demo works well and router works."

So from the main page
![image](https://user-images.githubusercontent.com/31169070/69482460-b1380100-0de9-11ea-956b-bc5874eb39e5.png)
if we select one item, it opens the item details.
![image](https://user-images.githubusercontent.com/31169070/69482462-bd23c300-0de9-11ea-90f2-b04bad26e71f.png)

But you see the URL link , it is the same. Also note than back button is disabled (as from browsers POV we are still in one page). This is the concern I raised yesterday. It means that we cannot share a URL and expect browser to redirect the user to that page, like a normal website would do. I will explore more examples and see if there is any other way.

**Comment by thevzurd** [comment]
![image](https://user-images.githubusercontent.com/31169070/69484589-f0268080-0e02-11ea-9d43-1dc64c0bca1f.png)

So there is no solution to it by default. I tried the solution at https://medium.com/flutter-community/more-than-a-flutter-web-app-is-a-full-flutter-website-c6bb210b1f16 , but it doesn't seem to work anymore. I raised a question in stack and flutter's discord channel. Meanwhile, I will also check other options. But I will stop looking for it, if that is something we cannot do right now
